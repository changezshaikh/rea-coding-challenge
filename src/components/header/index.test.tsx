import React from "react";
import { render } from "@testing-library/react";
import Header from ".";

describe("Header tests", () => {
  test("should render without crashing and match snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
