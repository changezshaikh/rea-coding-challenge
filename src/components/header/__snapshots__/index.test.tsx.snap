// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests should render without crashing and match snapshot 1`] = `
<div>
  <header>
    <div
      class="header__container"
    >
      <div
        class="l-padding"
      >
        <img
          alt="realestate.com.au Australia lives here"
          src="rea-logo.png"
        />
      </div>
    </div>
  </header>
</div>
`;
